docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -9,141 +9,125 @@ description: "A dual-region setup allows you to run Camunda in two regions synch
 | Connectors                     | If you are running Connectors and have a process with an inbound connector deployed in a dual-region setup, consider the following: <ul><li> when you want to delete the process deployment, delete it via Operate (not zbctl), otherwise the inbound connector won't deregister.</li><li>if you have multiple Operate instances running, then perform the delete operation in both instances. This is a [known limitation](https://github.com/camunda/camunda/issues/17762).</li></ul> |
 | Zeebe Cluster Scaling          | Not supported.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| Web-Modeler                    | Is a standalone component not covered in this guide. Modeling applications can operate independently outside of the automation clusters.                                                                                                                                                                                                                                                                                                                                                |
+| Web Modeler                    | Web Modeler is a standalone component that is not covered in this guide. Modelling applications can operate independently outside of the automation clusters.                                                                                                                                                                                                                                                                                                                           |
 
-### Platform considerations
+### Infrastructure and deployment platform considerations
 
-:::caution
-Multi-region setups have inherent complexities you should familiarize yourself with before choosing a dual-region setup.
-:::
+Multi-region setups come with inherent complexities, and it is essential to fully understand these challenges before selecting a dual-region configuration.
 
-For example, consider the following complexities (not covered in our guides):
+For example, consider the following areas that must be managed (Areas outside of our control and not covered in our guides):
 
 - Managing multiple Kubernetes clusters and their deployments across regions.
-- Monitoring and alerting.
+- Dual-Region monitoring and alerting.
```

Comment:
Whooot, you mean I have to monitor both regions? /s

I'm fine with the change, just found it funny that we have to point it out explicitly.

![](https://media.wired.com/photos/5f87340d114b38fa1f8339f9/master/w_1600%2Cc_limit/Ideas_Surprised_Pikachu_HD.jpg)

Reply:
The monitoring of a single cluster (how to) is our responsibility. So I qualify it that we don't deal with dual-region

Reply:
We tell people how to monitor and alert?
Isn't that very customer specific as there are different requirements and different tools with different solutions?

Reply:
Yes, we have information here and there, but general principle, we are/will/should telling customers how to monitor and what alerts to set up